worlds/oot_soh/location_access/overworld/graveyard.py

```diff
@@ -6,7 +6,166 @@
+    ])
+    # Connections
+    connect_regions(Regions.GRAVEYARD_HEART_PIECE_GRAVE, world, [
+        (Regions.THE_GRAVEYARD, lambda bundle: True)
+    ])
+
+    ## The Graveyard Composers Grave
+    # Locations
+    add_locations(Regions.GRAVEYARD_COMPOSERS_GRAVE, world, [
+        (Locations.GRAVEYARD_ROYAL_FAMILYS_TOMB_CHEST, lambda bundle: has_fire_source(bundle)),
+        (Locations.SONG_FROM_ROYAL_FAMILYS_TOMB, lambda bundle: can_use_projectile(bundle) or can_jump_slash(bundle)),
+        (Locations.GRAVEYARD_ROYAL_FAMILYS_TOMB_SUNS_SONG_FAIRY, lambda bundle: can_use(Items.SUNS_SONG, bundle))
+    ])
+    # Connections
+    connect_regions(Regions.GRAVEYARD_COMPOSERS_GRAVE, world, [
+        (Regions.THE_GRAVEYARD, lambda bundle: True)
+    ])
+
+    ## The Graveyard Dampes Grave
+    # Events
+    add_events(Regions.GRAVEYARD_DAMPES_GRAVE, world, [
+        (EventLocations.DAMPES_GRAVE_NUT_POT, Events.CAN_FARM_NUTS, lambda bundle: True),
+        (EventLocations.DAMPES_WINDMILL_ACCESS, Events.DAMPES_WINDMILL_ACCESS, lambda bundle: is_adult(bundle) and can_use(Items.SONG_OF_TIME, bundle))
+    ])
+    # Locations
+    add_locations(Regions.GRAVEYARD_DAMPES_GRAVE, world, [
+        (Locations.GRAVEYARD_HOOKSHOT_CHEST, lambda bundle: True),
+        (Locations.GRAVEYARD_DAMPE_RACE_FREESTANDING_PO_H, lambda bundle: is_adult(bundle) or can_do_trick(Tricks.GY_CHILD_DAMPE_RACE_POH, bundle)),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_POT1, lambda bundle: can_break_pots(bundle)),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_POT2, lambda bundle: can_break_pots(bundle)),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_POT3, lambda bundle: can_break_pots(bundle)),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_POT4, lambda bundle: can_break_pots(bundle)),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_POT5, lambda bundle: can_break_pots(bundle)),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_POT6, lambda bundle: can_break_pots(bundle)),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_RUPEE1, lambda bundle: True),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_RUPEE2, lambda bundle: True),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_RUPEE3, lambda bundle: True),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_RUPEE4, lambda bundle: True),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_RUPEE5, lambda bundle: True),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_RUPEE6, lambda bundle: True),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_RUPEE7, lambda bundle: True),
+        (Locations.GRAVEYARD_DAMPES_GRAVE_RUPEE8, lambda bundle: True)
+    ])
+    # Connections
+    connect_regions(Regions.GRAVEYARD_DAMPES_GRAVE, world, [
+        (Regions.THE_GRAVEYARD, lambda bundle: True),
+        (Regions.KAK_WINDMILL, lambda bundle: is_adult(bundle) and can_use(Items.SONG_OF_TIME)) # For some reason the original line has a false at the end when making the entrance. Not sure what is up with that: Entrance(RR_KAK_WINDMILL,  []{return logic->IsAdult && logic->CanUse(RG_SONG_OF_TIME);}, false),
+    ])
```

Comment:
I genuinely don't understand why it's on the ship side either, but I can't see why it would be relevant for this condition. So feel free to remove the comment and ignore it.